### PR TITLE
task(settings): do not show "take a photo" option on mobile

### LINKED ISSUE #5045: [avatar] Ensure mobile flow matches wires & functions as expected
The PR for this should at the very least make a tweak for displaying one "Upload" button if the user is on a mobile device rather than both "Upload" and "Take photo".

Ensure that on click, a native prompt asks the user if they'd like to take a new photo or use file upload. Double check this all works as expected, including removing the photo and rotating/zooming an uploaded photo, and make any needed tweaks for a smooth mobile experience.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-1717)
┆Issue Number: FXA-1717

### PR DESCRIPTION
**Pull request by dannycoates**
## Because

- the figma designs exclude the "Take photo" option on mobile

closes #5045

### CODE REVIEW

File: packages/fxa-settings/src/components/PageAvatar/index.tsx [hunk 1]
```diff
@@ -44,6 +44,7 @@ export const PageAddAvatar = (_: RouteComponentProps) => {
   const { avatar } = useAccount();
   const alertBar = useAlertBar();
   const [saveEnabled, setSaveEnabled] = useState(false);
+  const isMobile = /mobi/i.test(navigator.userAgent);
```
Comment (6a68) [inline packages/fxa-settings/src/components/PageAvatar/index.tsx:47]: Hmm, what about `isMobileDevice` in `lib/utilities`? Let's merge this check into that check or just prefer that check.

**Review by 6a68 — CHANGES_REQUESTED** [review]
LGTM, one question about where the mobile check should live

File: packages/fxa-settings/src/components/PageAvatar/index.tsx (hunk 1 above)
Comment (dannycoates) [reply packages/fxa-settings/src/components/PageAvatar/index.tsx:47]: Thanks! I didn't think to check if we had one already

Comment (dannycoates) [reply packages/fxa-settings/src/components/PageAvatar/index.tsx]: IDK how I feel about the combined approach but, meh 🤷

Comment (6a68) [reply packages/fxa-settings/src/components/PageAvatar/index.tsx]: lol, useragent sniffing is an ugly hack regardless, this LGTM

**Review by 6a68 — APPROVED** [review]

**Pull request merged**